managed/services/agents/vmagent.go

```diff
@@ -35,6 +36,8 @@ func vmAgentConfig(scrapeCfg string) *agentpb.SetStateRequest_AgentProcess {
 		maxScrapeSize = space
 	}
 
+	interfaceToBind := env.GetEnv("PMM_DEV_BIND_INTERFACE", "127.0.0.1")
```

Comment:
If we need this in multiple places I suggest to create function `GetDevInterface` or something like this in `envvars` package

Reply:
okay